openlibrary/templates/site/alert.html

```diff
@@ -4,7 +4,7 @@
 <div id="topNotice">
   <div class="page-banner page-banner-black page-banner-center">
     <div class="iaBar">
-      <div class="iaBarLogo">
+      <div class="iaBarDonate">
```

Comment:
This change is a bit confusing.  To my mind, the class name implies that this element is the donate button (instead of the logo).